### ISSUE TITLE
Return of Wii version

### ISSUE THREAD

**Issue by Edu789** [libretro/beetle-vb-libretro#62]
The Wii version is not coming in the nightly build ...

**Comment by twinaphex** [comment]
My bad, I had not noticed it was missing -

https://github.com/libretro/beetle-vb-libretro/commit/2999cceaeb04afb0e2d62b5c1e212f279d07001e

Hopefully within 6 hours it should be there again.

**Comment by Edu789** [comment]
Thanks

**Comment by Edu789** [comment]
https://github.com/libretro/RetroArch/issues/11217#issuecomment-785609663

The Fmsx and Cap32 are also gone and the Frodo and Dosbox hasn't been added yet...

**Comment by twinaphex** [comment]
Does beetle vb work again at least?

**Comment by twinaphex** [comment]
OK, I added fmsx and cap32 again.

**Comment by Edu789** [comment]
> Does beetle vb work again at least?

I haven't tested it yet.

**Comment by Edu789** [comment]
> OK, I added fmsx and cap32 again.

Thanks, i just need to add the frodo and dosbox now, on that link I have to send you these two.

**Comment by twinaphex** [comment]
Would appreciate if you could do that.

I also added frodo. So in tomorrow's nightlies, fmsx, cap32 and frodo should be in there. No idea if they work, you'll have to test that.

**Comment by Edu789** [comment]
> Does beetle vb work again at least?

Yes, it is working, but it has a sizzle in the audio, other colors like super nintendo are also with this problem ...

**Comment by Edu789** [comment]
> Would appreciate if you could do that.
> 
> I also added frodo. So in tomorrow's nightlies, fmsx, cap32 and frodo should be in there. No idea if they work, you'll have to test that.

**Comment by Edu789** [comment]
> Would appreciate if you could do that.
> 
> I also added frodo. So in tomorrow's nightlies, fmsx, cap32 and frodo should be in there. No idea if they work, you'll have to test that.

Frodo needs to map the virtual keyboard to start the game, it does not start automatically as explained in this link:
https://gbatemp.net/threads/wii-c64-core-frodo-with-some-improvements.574540/

**Comment by Edu789** [comment]
> Would appreciate if you could do that.
> 
> I also added frodo. So in tomorrow's nightlies, fmsx, cap32 and frodo should be in there. No idea if they work, you'll have to test that.

We just need to add the Dosbox now, it works but the classic control is not mapping the buttons.
